.gitmodules

```diff
@@ -1,49 +1,49 @@
 [submodule "proto"]
 	path = proto
-	url = ../firmware-protobuf.git
+	url = https://github.com/particle-iot/firmware-protobuf.git
 [submodule "third_party/nrf5_sdk/nrf5_sdk"]
 	path = third_party/nrf5_sdk/nrf5_sdk
-	url = ../nrf5_sdk.git
+	url = https://github.com/particle-iot/nrf5_sdk.git
 [submodule "third_party/lwip/lwip"]
 	path = third_party/lwip/lwip
-	url = ../lwip.git
+	url = https://github.com/particle-iot/lwip.git
 	branch = particle
 [submodule "third_party/lwip/lwip-contrib"]
 	path = third_party/lwip/lwip-contrib
-	url = ../lwip-contrib.git
+	url = https://github.com/particle-iot/lwip-contrib.git
 	branch = master
 [submodule "third_party/freertos/freertos"]
 	path = third_party/freertos/freertos
-	url = ../freertos.git
+	url = https://github.com/particle-iot/freertos.git
 	branch = particle
 [submodule "third_party/mbedtls/mbedtls"]
 	path = third_party/mbedtls/mbedtls
-	url = ../mbedtls.git
+	url = https://github.com/particle-iot/mbedtls.git
 	branch = particle-mbedtls-2.4.2
 [submodule "third_party/nanopb/nanopb"]
 	path = third_party/nanopb/nanopb
-	url = ../nanopb.git
+	url = https://github.com/particle-iot/nanopb.git
 	branch = particle-nanopb-0.3.9
 [submodule "third_party/openthread/openthread"]
 	path = third_party/openthread/openthread
-	url = ../openthread.git
+	url = https://github.com/particle-iot/openthread.git
 [submodule "third_party/nrf_802154/nrf_802154"]
```

Comment:
We can also remove this one, as we are using the radio driver that is bundled with OpenThread.

Reply:
Yeah, not the `openthread` :laughing: We need it for mesh communications. I was talking about `[submodule "third_party/nrf_802154/nrf_802154]`.

Reply:
:man_facepalming: